Umweltzone/src/main/res/values-nl/strings.xml

```diff
@@ -0,0 +1,348 @@
+        Een Android-app voor milieuzones en dieselverboden in Duitsland
+    </string>
+
+    <!-- AboutActivity / Copyright -->
+    <string name="appinfo_copyright_title">Idee en realisatie</string>
+    <string name="appinfo_copyright_content">
+        (&#169;) Tobias Preuß, 2013 &#8211; 2019
+    </string>
+
+    <!-- AboutActivity / Description -->
+    <string name="appinfo_description_title">Beschrijving</string>
+    <string name="appinfo_description_content">
+        Deze applicatie toont de locatie van zones met lage emissie (LEZ) en Diesel
+        verbodszones (DPZ) en biedt meer informatie over dit onderwerp.
+        Rijden in een LEZ is beperkt tot voertuigen die zijn getagd met een
+        specifieke sticker. De sticker komt overeen met het emissieniveau van de
+        voertuig. Voertuigen die niet voldoen aan een bepaalde dieseluitlaatemissie
+        standaard zijn beperkt om een ​​DPZ in te voeren. Het doel van de inreisbeperkingen is
+        om de luchtkwaliteit in steden te verbeteren.
+        Fijne deeltjes moeten worden verminderd om de gezondheid van de burger te behouden. &#8212;
+        Naast de kaart en een lijst met steden is er een detailscherm dat
+        bevat individuele informatie voor de geselecteerde stad.
+        Veelgestelde vragen zijn te vinden via de
+        &#8220; FAQ &#8221; menu onderdeel. Bovendien zijn er links naar websites die
+        meer informatie bieden over LEZ, DPZ en gerelateerde onderwerpen.
+    </string>
+
+    <!-- AboutActivity / Disclaimer -->
+    <string name="appinfo_disclaimer_title">" Gegevens, kwaliteit en updates"</string>
+    <string name="appinfo_disclaimer_content">
+        Overal waar gegevens en coördinaten van de zones digitaal beschikbaar waren
+        formaat werden ze rechtstreeks opgehaald uit steden of overheden.
+        Verder was OpenStreetMap sinds een aantal een belangrijke bron
+        zones met lage emissie zijn daar te vinden. Bovendien hebben sommige zones
+        handmatig gedigitaliseerd op basis van de officiële publicaties van de
+        Federaal Milieuagentschap.
+        Vanwege het feit dat veel autoriteiten afbeeldingen met een lage resolutie publiceren
+        van gescande kaarten kan niet worden gegarandeerd dat de zone precies grenst
+        overeenkomen met het pad in de realiteit. Zonegegevens en coördinaten verschillen per bron,
+        kwaliteit en precisie in het digitalisatieproces.
+        Bovendien is er geen automatisme om op de hoogte te worden gebracht van aangebrachte wijzigingen
+        door de steden. Zonegegevens worden handmatig bijgewerkt en zijn sterk afhankelijk van de
+        feedback van vriendelijke gebruikers zoals jij.
+        Om deze redenen kunnen we niet garanderen voor een perfecte visualisatie en
+        wij zijn niet aansprakelijk voor kosten of straffen. We raden ten zeerste aan om altijd
+        vertrouw op het verkeersbord in de betreffende steden.
+    </string>
```

Comment:
Please check preview of the longer texts block in Android Studio in `activity_about.xml`. There is a couple of blanks in the texts. Scroll down in the preview to see the text.